src/app/pages/wallets/wallet-details/transaction-editor/pages-wallet-transaction-editor.component.ts

```diff
@@ -0,0 +1,100 @@
+import { Component, Inject } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { MatDialogRef, MAT_DIALOG_DATA } from '@angular/material/dialog';
+import { TServerEntityId } from 'src/app/common/http/common.http.types';
+import { checkInputError } from 'src/app/common/utils/forms/common-utils-forms-form-utils';
+import { WalletTransactionType } from 'src/app/domains/transactions/domains.transactions.constants';
+import { WalletsDetailsTransaction } from '../pages-wallet-details-item.model';
+import { IWalletTransactionModalFormType } from './pages-wallet-transaction.editor.types';
+
+export const regexAmount = /^\d+(\.\d{1,2})?$/;
+export const regexDate = /\d{2}\/\d{2}\/\d{4}/;
```

Comment:
For date you probably can't go for regex as this is not a string but `Date` object. Probably you need to go with something like this: https://www.concretepage.com/angular-material/angular-material-datepicker-format